WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -1233,6 +1285,9 @@ extension NotificationDetailsViewController {
 
         onSelectedNoteChange?(previous)
         note = previous
+        removeConfettiView()
+        addConfettiViewIfNeeded()
+        showConfetti()
```

Comment:
Since this block of methods is used in a few places how do you feel about  combining `removeConfettiView()` and `addConfettiViewIfNeeded()` into the `showConfetti()` method and rename it to `showConfettiIfNeeded()` ?